phpcs.ruleset.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
+<ruleset name="WordPress Coding Standards for Performance related Plugins">
+	<description>Sniffs for WordPress plugins</description>
+
+	<rule ref="PHPCompatibility"/>
+	<config name="testVersion" value="7.0-"/>
+
+	<rule ref="WordPress-Docs"/>
+	<rule ref="WordPress-Extra" />
+	<rule ref="WordPress.WP.I18n"/>
+
+	<arg value="ps"/>
+	<arg name="extensions" value="php"/>
+
```

Comment:
```suggestion
<arg name="parallel" value="20"/>

```
Would be great to include this to speed up linting. See https://github.com/WordPress/performance/pull/953/files#r1469088154